packages/node_modules/@webex/internal-plugin-encryption/src/encryption.js

```diff
@@ -72,28 +72,16 @@ const Encryption = WebexPlugin.extend({
     }
 
     const shunt = new EventEmitter();
-    const promise = this._fetchDownloadUrl(scr)
-      .then((uri) => {
-        const options = {
-          method: 'GET',
-          uri,
-          responseType: 'buffer'
-        };
-
-        const ret = this.request(options);
-
-        transferEvents('progress', options.download, shunt);
-
-        return ret;
-      })
-      .then((res) => this.decryptBinary(scr, res.body));
+    const promise = this._fetchDownloadUrl(scr);
 
     proxyEvents(shunt, promise);
 
     return promise;
   },
 
   _fetchDownloadUrl(scr) {
+    const _this2 = this;
```

Comment:
I see `_this2` is [called later](https://github.com/webex/webex-js-sdk/pull/1377/files#diff-22291d397df84ad20147dd45e8209ce4R106) when it could be called as `this`.